atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -63,17 +63,16 @@ def generate_cachito_env_file(self):
         cachito request.
         """
 
-        if self.buildargs:
-            self.log.info('Creating %s file with environment variables '
-                          'received from cachito request', CACHITO_ENV_FILENAME)
-
-            # Use dedicated dir in container build workdir for cachito.env
-            abs_path = os.path.join(self.workflow.builder.df_dir,
-                                    self.REMOTE_SOURCE, CACHITO_ENV_FILENAME)
-            with open(abs_path, 'w') as f:
-                f.write('#!/bin/bash\n')
-                for env_var, value in self.buildargs.items():
-                    f.write('export {}={}\n'.format(env_var, quote(value)))
+        self.log.info('Creating %s file with environment variables '
+                      'received from cachito request', CACHITO_ENV_FILENAME)
```

Comment:
> received from cachito request

Is this description correct? The `self.buildargs` is described in the `__init__` docstring that it is the container build args.

Reply:
I believe so. Here is a snippet of ARGs added to container build Dockefile:
* http://pastebin.test.redhat.com/923283